x-pack/plugins/observability_solution/observability/server/ui_settings.ts

```diff
@@ -528,7 +528,7 @@ export const uiSettings: Record<string, UiSettings> = {
     name: i18n.translate('xpack.observability.profilingAWSCostDiscountRateUiSettingName', {
       defaultMessage: 'AWS EDP discount rate (%)',
     }),
-    value: '0',
+    value: 0,
```

Comment:
@cauemarcondes I see you added these settings as strings, can you weigh in on whether it's ok to update them to numbers here like this? If so, I think we can appove this PR. Thanks!

Reply:
I confirmed with Caue that this was done only to get around the 0/falsy bug that this PR fixes. Changes LGTM!